www-servers/nginx-unit/nginx-unit-1.4.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python2_7 python3_{3,4,5,6})
+
+inherit python-single-r1
+
+MY_P="unit-${PV}"
+DESCRIPTION="A dynamic web and application server"
+HOMEPAGE="https://unit.nginx.org"
+SRC_URI="https://unit.nginx.org/download/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+MY_IUSE="perl python ruby"
+MY_USE_PHP="php5-6 php7-0 php7-1 php7-2"
+IUSE="${MY_IUSE} ${MY_USE_PHP}"
```

Comment:
I think `IUSE=php` as global control is the current practice.

Reply:
I'm not certain I understand what you mean? As described in the linked bug report, the user needs to explicitly specify for which PHP versions support modules need to be built.